src/forms/BoundForm.stories.tsx

```diff
@@ -162,7 +168,7 @@ const genres: NestedOption<HasIdAndName>[] = [
 ];
 
 type AuthorInput = BaseAuthorInput & {
-  iconCardSelection?: boolean | null;
```

Comment:
While the codegened input types generally have all keys as optional, it's common for implementors to extend those types with known required keys in the form component. This was causing a type error since the `reactNode*` mapped type was not also wrapped in `Partial<>`

Reply:
Ah, so the lack of `?` is specifically to cover the mapped type regressing? That makes sense/is a great idea; maybe leave an in-source comment so a future maintainer doesn't wonder why its missing, and puts it back.